nameable.rb

```diff
@@ -0,0 +1,5 @@
+class Nameable
+  def correct_name
+    raise NotImplementedError
+  end
+end
```

Comment:
- [ ] [REQUIRED] Good job so far! It would be nice If you implement your `def` value like that:

```ruby
class Nameable
    def correct_name
        raise NotImplementedError, "#{self.class} has not implemented method '#{__method__}'"
    end
end
```

Reply:
Ok, thanks